src/CSharp/CodeCracker/Style/PropertyPrivateSetAnalyzer.cs

```diff
@@ -39,7 +39,7 @@ private static void AnalyzeClass(SyntaxNodeAnalysisContext context)
             var invocationExpression = (PropertyDeclarationSyntax)context.Node;
             var semanticModel = context.SemanticModel;
 
-            if (invocationExpression.AccessorList.Accessors.Count == 1) return;
+            if (invocationExpression.AccessorList == null || invocationExpression.AccessorList.Accessors.Count == 1) return;
```

Comment:
@carloscds please, see my test case. In C# 6 we have expression bodied properties, like `public int MyProperty => 0;`. They are getter-ony properties and don't have a get acessor.